source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,103 @@
+
+1. Sign in to your ePDQ account.
+1. Go to Configuration then Payment methods.
+1. Select 3D Secure.
+1. Check that 3DS is on.
+1. In the GOV.UK Pay admin tool, go to your My services page, then select the account you want to set up
+1. In the settings, select the View link under 3D Secure.
+1. Select On.
 
-1. Select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-2. In the settings, select the __View__ link under __3D Secure__.
-3. Select __On__.
-4. Copy your 3DS Flex details into the [PSP settings in your GOV.UK Pay account](https://selfservice.payments.service.gov.uk/your-psp).
-5. Check your account still works by [making a payment with a real payment card](/switching_to_live/#6-check-your-live-account-works).
+___OR___
 
-## ePDQ
+GOV.UK Pay has built the integration with 3DS2 for ePDQ. You do not need to make any changes to your integration. ePDQ does not apply any exemptions (LINK).
 
-1. Sign in to your [ePDQ account](https://payments.epdq.co.uk/Ncol/Prod/BackOffice/login/index).
-2. Go to __Configuration__ then  __Payment methods__.
-3. Select __3D Secure__.
-4. Check that 3DS is on.
-5. In the GOV.UK Pay admin tool, go to your [__My services__](https://selfservice.payments.service.gov.uk/my-services) page, then select the account you want to set up
-6. In the settings, select the __View__ link under __3D Secure__.
-7. Select __On__.
+## Set up SCA with Smartpay
 
-## SmartPay
+Smartpay does not support 3DS2. You can only enable 3DS.
 
-1. Select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-2. In the settings, select the __View__ link under __3D Secure__.
-3. Select __On__.
+1. Select the account you want to set up on the My services page in the GOV.UK Pay admin tool.
+1. In the settings, select the View link under 3D Secure.
+1. Select On.
 
 You do not need to do anything in your Smartpay account.
+
+## Exemptions to 3DS2
+
+3DS2 does not apply to:
+
+- transactions where either the merchant or payer is outside the EEA
+- merchant initiated payments such as subscriptions
+- telephone (MOTO) payments
+
+The following payment types may be exempt from 3DS2 depending on your PSP:
+
+- low value transactions, for example for those under £30
+- low risk transactions when yout PSP has low fraud levels across its platform
+- corporate payments, unless the corporate card is in the name of an individual
+
+## Questions
+
+___QP: Are there any exemptions for Stripe___
+
+___QP: What are the exact dates that you need to set up SCA by?___
+
+___QP: Do you need to set anything up with ePDQ or not?___
+
+___QP: Are there any exemptions for ePDQ?___
+
+___QP: Any trnx that 3DS1 does not apply to?___
+
+___QP: If you're a service, do you have to set up 3DS2 or can you choose to set up 3DS instead? Does that count as SCA?___
```

Comment:
it does count as SCA. a service doesn't have to switch (and in the case of smartpay, they can't, because we're not investing more in that integration). 
but 
- 3ds won't be supported indefinitely (though timeline is not clear)
- 3ds transactions will incur additional costs (mastercard and visa are adding charges to 3ds transactions to encourage merchants to move to 3ds2)
- 3ds provides more friction for users, it's basically a less good experience all round